frontend/form-builder/src/components/FormSettings/RadioSettings.tsx

```diff
@@ -1,29 +1,27 @@
 import React, { useContext } from "react";
 import { useTranslation } from "react-i18next";
 import { Textarea, Textfield } from "@digdir/design-system-react";
-import { ButtonGroup } from "./ButtonGroup";
 import { FormBuilderContext } from "../../context";
 
 export const RadioSettings = () => {
-  const { currentComponent } = useContext(FormBuilderContext);
+  const { selectedItem } = useContext(FormBuilderContext);
```

Comment:
Here you can also use destructuring since it is only one level that is used: 

`const { name, label = "", radioChoices= "" } = useContext(FormBuilderContext).selectedItem ;`